### PR TITLE
List enrichment entries by type

### PR DESCRIPTION
**Pull request by ruffle1986**
This PR is about the first iteration of listing enrichment entries based on a given type.

It introduces the enrichment-entries module which is responsible for displaying the entries based on a given type. For now, I've used a very basic table component to display the basic attributes for each entry.

Now we have for basic attributes (key, timestamp, user, value) declared in the EnrichmentEntry model. This is extendable of course. 

Testing
=====

To get to the entries screen, type `/enrichment-entries/type` in your browser's address bar. The type part can be anything for now. Later on, we can get the exact type value from the query string and request the entries with the given value.

The list consists of one item which is also hard coded until we have a real data source up and running.

### CODE REVIEW

File: src/app/app.module.ts [hunk 1]
```diff
@@ -6,6 +6,8 @@ import { StoreDevtoolsModule } from '@ngrx/store-devtools';
 import { environment } from '../environments/environment';
 import { StoreModule, MetaReducer } from '@ngrx/store';
 import { storeFreeze } from 'ngrx-store-freeze';
+import { EnrichmentEntriesModule } from './enrichment-entries/enrichment-entries.module';
```
Comment (sardell) [inline src/app/app.module.ts]: https://angular.io/guide/styleguide#import-line-spacing

We really should use TSLint to enforce the order and spacing of our imports. Ordering and spacing makes it much easier to parse the imports as the list expands. I'm fine with this being done in another PR since this is sort of out-of-scope here.

Here's a TSLint rule for it: https://palantir.github.io/tslint/rules/ordered-imports/

Comment (ruffle1986) [reply src/app/app.module.ts]: I'm in. Especially because it has a fixer.

**Comment by ruffle1986** [comment]
This is going to be next. Very good idea!
